contributingGuides/FORMS.md

```diff
@@ -198,6 +198,38 @@ function onSubmit(values) {
 </Form>
 ```
 
+`Form.js` also works with inputs component that's nested on a component that doesn't wrap with any HOC.
```

Comment:
Can we simplify this statement? Even break it into two lines if required. 

`Form.js` also works with inputs from a nested component. The only exception is that the nested component shouldn't be wrapped around any HoC.

@luacmartins can also share thoughts on this one.

Reply:
>Form.js also works with inputs from a nested component. The only exception is that the nested component shouldn't be wrapped around any HoC.

I'm onboard with your suggestion.